### ISSUE TITLE
[Bug] CRT shader doesn't work on HTML exports with `alone=1`

### ISSUE THREAD

**Issue by scambier** [nesbox/TIC-80#2229]
Version 1.1.2624-dev

When exporting an html with the option `alone=1`, the CRT shader doesn't work.
It prints these error messages non-stop:
```
Failed to load vertex shader: [tic80.js:8:23405](http://127.0.0.1:8080/tic80.js)
Error: vertex shader is empty. [tic80.js:8:23405](http://127.0.0.1:8080/tic80.js)
Error: pixel shader is empty.
```

**Comment by nesbox** [comment]
can't reproduce on the latest dev

**Comment by scambier** [comment]
Found the issue: the CRT shader works for Lua carts, but not JS (and maybe others) when using `alone=1`

I guess the CRT code is somehow bounded to Lua, and gets stripped with `alone=1`


- Works: [html-test-lua.zip](https://github.com/nesbox/TIC-80/files/12385417/html-test-lua.zip)
- Doesn't work: [html-test-js.zip](https://github.com/nesbox/TIC-80/files/12385418/html-test-js.zip)
- Works: [html-test-js-not-alone.zip](https://github.com/nesbox/TIC-80/files/12385416/html-test-js-not-alone.zip)

**Comment by nesbox** [comment]
I moved the shader code from config directly to `main.c`, this allows us to init CRT effect without Lua.
fixed here 5b9739967a4fd07a035f72b45133d400a93eb3f5
